src/main/resources/hudson/plugins/matrix_configuration_parameter/MatrixCombinationsParameterDefinition/index.groovy

```diff
@@ -1,13 +1,10 @@
 package hudson.plugins.matrix_configuration_parameter.matrixcombinationparameterDefinition
 
-import hudson.matrix.AxisList
-import hudson.matrix.Combination
-import hudson.matrix.Layouter
-import hudson.matrix.MatrixBuild
-import hudson.matrix.MatrixProject
+import hudson.matrix.*
+import hudson.model.Result
+import hudson.plugins.matrix_configuration_parameter.MatrixCombinationsParameterDefinition
```

Comment:
You may minimize changes in pull requests not to conflict with other pull requests.
(I don't think you need to update and fix this pull request as there's no other pull requests)